pkgs/tools/archivers/rpm2targz/default.nix

```diff
@@ -0,0 +1,69 @@
+{ bzip2
+, coreutils
+, cpio
+, fetchurl
+, gnutar
+, gzip
+, lib
+, stdenv
+, xz
+, zstd
+}:
+
+let
+  shdeps = [
+    bzip2
+    coreutils
+    cpio
+    gnutar
+    gzip
+    xz
+    zstd
+  ];
+
+in stdenv.mkDerivation rec {
+  pname = "rpm2targz";
+  version = "9.0.0.5g";
+
+  # git repo: https://gitweb.gentoo.org/proj/rpm2targz.git/
+  src = fetchurl {
+    url = "mirror://gentoo/distfiles/c0/${pname}-${version}.tar.xz";
+    sha256 = "Qye33TC/VeRzGYREuSACliSUjiSvV17f+c5Gbc1AdeA=";
+  };
+
+  patches = [
+    ./rpm2targz-9.0.0.5g-add-zstd-support.patch
+    ./rpm2targz-9.0.0.5g-add-zstd-support-2.patch
```

Comment:
Can we use fetchpatch?

Reply:
The first change is directly from the gentoo repo, it would be gone if the gentoo repo changes, and is afaik not even present inside of the "gentoo/proj/rpm2targz" repository. The second patch is adjusted and included by myself, to make the tool more symmetrical in regards to zstd.

I could however (and probably will, but it might take a while) contact the maintainer of the Gentoo fork, https://wiki.gentoo.org/wiki/User:Vapier, and ask for inclusion of the patches in the Gentoo fork repo.

Reply:
ok, I sent an email.

Reply:
ok, got a new release: https://dev.gentoo.org/~vapier/dist/rpm2targz-2021.03.16.tar.xz